src/reference/sectionA_definitions.inc

```diff
@@ -146,13 +156,6 @@ Non-specific information items
       changing the number of whitespace characters in such an information item,
       or changing the number of whitespace characters at the beginning or end of any character information item.
 
-.. marker_cellml_information_sets_3
```

Comment:
removing this marker will need a change elsewhere to match right? or is this one not used directly @kerimoyle ?

Reply:
Fixed them!